Playlist_Genre_Jazz.php

```diff
@@ -44,7 +44,7 @@
 
```

Comment:
all genres playlist are copy and pasted the same to save time; besides from queries